content/09.evolution.md

```diff
@@ -168,7 +171,9 @@ Moderna recently published data for their mRNA-1273 vaccine that showed no signi
 On the other hand, Moderna reported a reduced but significant neutralization against the B.1.351 variant upon vaccination [@doi:10.1101/2021.01.25.427948].
 Indeed, Pfizer–BioNTech reported that sera from twenty participants vaccinated with the BNT162b COVID-19 vaccine in previous clinical trials [@doi:10.1056/NEJMoa2034577; @doi:10.1056/NEJMoa2027906] elicited equivalent neutralizing titers against isogenic Y501 SARS-CoV-2 on an N501Y genetic background _in vitro_ [@doi:10.1101/2021.01.07.425740].
 Another study has reported that the plasma neutralizing activity against SARS-CoV-2 variants encoding the combination of K417N:E484K:N501Y or E484K or N501Y was variably and significantly reduced in the sera of twenty participants who received either the Pfizer–BioNTech BNT162b (n = 6) vaccine or the Moderna’s mRNA-1273 vaccine (n =14) [@doi:10.1101/2021.01.15.426911]. 
-For now, the consensus appears to be that the FDA approved vaccines still seem to be effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a slightly lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948]. 
+In a study focusing on serum samples from a combination of convalescent individuals, those who obtained the mRNA-1273 vaccine, and those who obtained Novavax, in comparison to the D614G variant, the B.1.419 variant was 2-3 times less sensitive to neutralization while the B.1.351 variant was 9-14 times less sensitive [@doi:10.1056/NEJMc2103740].
+Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines [@doi:10/f53t; @doi:10.1038/s41586-021-03412-7; @doi:10.1056/NEJMc2103740]. 
+For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally to modestly effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
```

Comment:
```suggestion
For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
```

Reply:
I picked generally over modestly as its probably too early to quantify the effectiveness